numbert/utils/data_utils/tf_dl.py

```diff
@@ -20,19 +20,28 @@
 import tensorflow as tf
 tf.enable_eager_execution()
 import torch
-import numpy as np
 
 class TFRecordDataLoader(object):
-    def __init__(self, records, batch_size, max_seq_len, train, num_workers=2, seed=1, threaded_dl=False):
+    def __init__(self, records, batch_size, max_seq_len, train, num_workers=2, seed=1, threaded_dl=False, task="msmarco"):
         tf.set_random_seed(seed)
         if isinstance(records, str):
             records  = [records]
 
-        self.record_converter = Record2Example({"input_ids": tf.FixedLenFeature([max_seq_len], tf.int64),
-                                                "attention_mask": tf.FixedLenFeature([max_seq_len], tf.int64),
-                                                "token_type_ids": tf.FixedLenFeature([max_seq_len], tf.int64),
-                                                "labels" : tf.FixedLenFeature([1], tf.int64),
-                                                "guid" : tf.FixedLenFeature([1], tf.int64)})
+        if task == "treccar":
+            self.record_converter = Record2Example({"input_ids": tf.FixedLenFeature([max_seq_len], tf.int64),
+                                                    "attention_mask": tf.FixedLenFeature([max_seq_len], tf.int64),
```

Comment:
Indentation seems wrong here

Reply:
I think this indentation matches that in the original implementation of tf_dl in the Megatron_LM repository. Also, I think that the next line is aligned properly, so not sure how this is wrong. I'm pretty new to code review on GitHub, so it is very possible I'm missing something  :)

Reply:
Oh, I see what happened, git broke the line when showing it. It is fine. We could use the 80-char limit but since we are not following PEP8 convention it is fine to have longer lines.

Reply:
Yeah, I tend to prefer that too but all of Megatron_LM and transformers repositories seem to be created without this 80-char limit.